react/components/ErrorPage/messages/index.ts

```diff
@@ -0,0 +1,34 @@
+import bg from './bg.json'
+import de from './de.json'
+import en from './en.json'
+import es from './es.json'
+import fr from './fr.json'
+import it from './it.json'
+import ja from './ja.json'
+import ko from './ko.json'
+import nl from './nl.json'
+import pt from './pt.json'
+import ro from './ro.json'
+import th from './th.json'
+
+export const messages: Record<string, Record<string, string>> = {
+  'en-US': en,
+  'es-AR': es,
+  'fr-FR': fr,
+  'pt-BR': pt,
+  'ja-JP': ja,
+  'ko-KR': ko,
+  'it-IT': it,
+  'nl-NL': nl,
+  'ro-RO': ro,
+  'bg-BG': bg,
+  'th-TH': th,
+  'de-DE': de,
+}
+
+export function getMessages(locale: string) {
+  const localeMessages = (locale && messages[locale]) || {}
+  const fallbackMessages = messages['en-US']
+
+  return { ...fallbackMessages, ...localeMessages }
```

Comment:
```suggestion
  const fallbackMessages = messages['en-US']
  const localeMessages = (locale && messages[locale]) || fallbackMessages

  return localeMessages
```

Just a suggestion, but it's up to you, to me this way looks more easy to read

Reply:
Sounds good I'll change it

Reply:
Actually thinking a bit more, that destructing I did on line 33 was actually intentional, to make sure that even if we DO have messages (for examples for french) but only SOME keys are missing. So in this case of incomplete messages there would be no risk of some messages showing up as "render/error.message-name", the missing messages would show up in english instead

Reply:
Nowadays all messages are there for all mapped languages, right? I'm not sure if this is a real problem that we should worry about 🤔

Reply:
I feel like it's safer to have the fallback still. More messages could be added with time, and I've seen it happen a few times if there are new feature in this page. Situations like clients seeing messages like "render/error.message-name" in production.